invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_SEARCH_CLASS = "invenio_search:RecordsSearch"
+
+ORCID_RECORDS_PID_TYPE = 'literature'
```

Comment:
Can it be solved with only `ORCID_RECORDS_PID_FETCHER` that can fetch both types and `ORCID_RECORD_SEARCH_CLASS`?